packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -17,8 +17,13 @@ import 'theme_data.dart';
 /// such an image, the user's initials. A given user's initials should
 /// always be paired with the same background color, for consistency.
 ///
+/// If [foregroundImage] fails then [backgroundImage] is used. If that fails
+/// too, [backgroundColor] is considered.
```

Comment:
At line 21, I think `is considered` is not really a fit here, maybe use `is used` also. Correct me if I am wrong 🙈.

Reply:
Any suggestions?

Reply:
I mean line 21 to be like this:
```
/// too, [backgroundColor] is used.
```

considered is like it may be used but since it is fallback it will be used no matter what.

Reply:
Thank you!